frontend/public/components/nav.jsx

```diff
@@ -380,6 +380,7 @@ export class Nav extends React.Component {
           </NavSection>
 
           <NavSection required={FLAGS.OPERATOR_LIFECYCLE_MANAGER} text="Operators" img={operatorImg} activeImg={operatorActiveImg} >
+            <HrefLink href="/marketplace" name="Kubernetes Marketplace" activePath="/marketplace/" onClick={this.close} />
```

Comment:
Do we need a separate flag to know if marketplace is enabled? If there is a CRD we can check for, we can add it to the list here:

https://github.com/openshift/console/blob/master/frontend/public/features.ts#L57-L64

then use `required={FLAGS.MARKETPLACE}` to show the nav only when the flag is enabled